<launch>
  <node name="extract" pkg="image_view" type="extract_images" respawn="false" output="screen" cwd="ROS_HOME">
    <remap from="image" to="/camera_front/image_raw"/>
    <param name="image_transport" value="compressed"/>
  </node>
</launch>
